src/appmixer/intelswift/core/HookCreate/HookCreate.js

```
@@ -0,0 +1,33 @@
+        const { platformType, syncId, name, firstName, lastName, tenantId, projectUUID } = context.messages.in.content;
+
+        // remaining to handles tenantId and projectId
+
+        let body = {};
+        if(!platformType) throw 'platformType not found';
+
+        if(platformType == 'zoho') body.zohoId = syncId;
+        if(platformType == 'hubspot') body.hubspotId = syncId;
+        if(platformType == 'pipedrive') body.pipedriveId = syncId;
+
+        const { data } = await makeRequest(context, `/api/v1/marketplace/create/contact`, { 
```

ESLint

[error] Strings must use singlequote.

(quotes)